.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bugs
+about: Crashes and other bugs
+title: ''
+labels: 'bug'
+assignees: ''
```

Comment:
Nit: can we omit `assignees`?

Reply:
Oh looks like you can omit them. Removed.